schemas/v3.1/schema.json

```
@@ -774,7 +774,7 @@
         }
       },
       "patternProperties": {
-        "^[1-5][0-9X]{2}$": {
+        "^[1-5](?:[0-9]{2}|XX)$": {
```

```suggestion
        "^[1-5](?:[0-9][0-9]|XX)$": {
```

this might be a bit more readable IMHO

---

ok, taken.